components/mixins/profile/b2b-person-details.schema.json

```diff
@@ -0,0 +1,216 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/mixins/b2b-person-details",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "XDM Business Person Details",
+  "type": "object",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "description": "This mixin is used to capture B2B Person specific attributes. A B2B Person could be a contact/lead etc.",
+  "definitions": {
+    "b2b-person-details": {
+      "properties": {
+        "xdm:personType": {
+          "title": "Person Type",
+          "description": "Type of the B2B person.",
+          "type": "string"
+        },
+        "xdm:personStatus": {
+          "title": "Person Status",
+          "description": "Field recording the current marketing/sales status of the Person.",
+          "type": "string"
+        },
+        "xdm:personScore": {
+          "title": "Person Score",
+          "description": "A Score generated for a person by the CRM system.",
+          "type": "number"
+        },
+        "xdm:personGroupID": {
+          "title": "Person Group ID",
+          "description": "Group identifier for the person.",
+          "type": "number"
+        },
+        "xdm:accountID": {
+          "title": "Contact Account id",
+          "description": "ID of the account that’s the parent of this person.",
+          "type": "string"
+        },
+        "xdm:personOwnerID": {
+          "title": "Person Owner ID",
+          "description": "Person owner id.",
+          "type": "string"
+        },
+        "xdm:personEmail": {
```

Comment:
We already have a standard field for Profile named xdm:personalEmail that comes from the profile-personal-details mixin. This field should be renamed to "xdm:personalEmail" or you could already reference and use that mixin.

Reply:
I did notice this so, I removed this already.